lib/handlers/random/index.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const Sentiment = require('sentiment');
+const Adjectives = require('./adjectives.json');
+const Names = require('./names.json');
+
+
+const internals = {
+  randomName: () => {
+    const index = Math.floor(Math.random() * Names.length);
+    return Names[index];
+  },
+  randomAdjective: () => {
+    const index = Math.floor(Math.random() * Adjectives.length);
+    return Adjectives[index];
+  },
+  safeName: () => {
+    const name = internals.randomName();
+    const adjective = internals.randomAdjective();
+
+    const str = `${adjective}-${name}`;
```

Comment:
If we stored the adjective and name list with spaces instead of hyphens, we could simplify this code:

```js
const str = `${adjective} ${name}`;

if (Sentiment(str).score < 0) {
  return internals.safeName();
}

return str.join('-');
```

That would remove the `split(/-/)` completely, and only perform the `join(' ')` with a happy name. If storing the lists with spaces is an issue, we could do a hyphen to space replacement at server startup.

Side note: Maybe we should monitor any unhappy name creation to see if we can avoid the recursive case.